Dockerfile

```diff
@@ -11,11 +11,20 @@ RUN apt-get update \
     && apt-get -qy upgrade --fix-missing --no-install-recommends \
     && apt-get -qy install --fix-missing --no-install-recommends \
         curl \
+        git \
```

Comment:
:+1: indeed, having one way to install all dependencies for Invenio for both Docker and non-Docker development environments sounds like a really good solution we should try to achieve at some point :)